modules/oe_theme_helper/templates/oe-theme-helper-address-inline.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Default template for the 'oe_theme_helper_address_inline' address formatter.
+ *
+ * Available variables:
+ *   - address: Address object.
+ *   - address_items: Address items.
+ *   - address_delimiter: Delimiter between address items.
+ *
+ * @ingroup themeable
+ */
+#}
+<p class="address" translate="no">
```

Comment:
Make this a `span` and this is an inline formatter, and remove the class name, just ` <span translate="no">...`